### PR TITLE
Add checks to find out reason of #2262

### PR DESCRIPTION
**Pull request by Pirulax**
Regarding the crash at #2262

I've looked at the models, but I didnt find any frame with name longer than 24 chars.
I'm adding this to see if this is the reason of the crashes.

Ah, also, function signature was wrong for the callback, so I fixed that.

### CODE REVIEW

File: Client/game_sa/CFileLoaderSA.cpp [hunk 1]
```diff
@@ -43,12 +43,42 @@ class CDamagableModelInfo
...
+    {
+        outDamage = false;
+        if (EndsWith("_l0") || EndsWith("_L0"))
+            TerminatedCopy(sizeof("_l0") - 1);
+        else {
+            assert(nodeNameLen < BuffSize);
+            strcpy_s(outName, nodeName);
+        }
+    }
 }
```
Comment (Pirulax) [inline Client/game_sa/CFileLoaderSA.cpp:82]: This is the POI (point of interest)

**Comment by Pirulax** [comment]
Actually, now that I looked at the code in the IDB, it seems like `bool` is the right type.

**Comment by Dutchman101** [comment]
It seems this was forgotten about, PR's with debug code for temporary logging purposes don't need the same code review attention as features or actual changes would. I feel like we should merge this as soon @Pirulax comments to confirm he's available for the process (that includes intepreting the results of debug logging, and indicating how to fix it). Since i know he's on his regular development activity break..

**Comment by Dutchman101** [comment]
Closing this in favour of [comment #4 on issue 2262](https://github.com/multitheftauto/mtasa-blue/issues/2262#issuecomment-997209318), as the crash cause was already discovered. In case you needed to collect even more info anyways, feel free to reopen

**Comment by Pirulax** [comment]
This also was a fix for the issue though, not just logging. I'll modify the code a little, and reopen the PR.

**Pull request closed without merging**